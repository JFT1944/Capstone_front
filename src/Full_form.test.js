
import React from "react";
import { render, fireEvent, getByTestId } from "@testing-library/react";
import FullForm from "./Full_Form";
import { BrowserRouter, MemoryRouter } from "react-router-dom";


// write a smoke test for the home component

it('should appear in a smoketest', () => {
    render(<BrowserRouter>
    <FullForm user={'user'} token={'token'}/>
    </BrowserRouter>)
})

it('should match snapshot', () => {
    const {asFragment} = render(<MemoryRouter><FullForm user={'user'} setUser={'setUser'} token={'token'} setToken={'setToken'}/></MemoryRouter>)
    expect(asFragment()).toMatchSnapshot()
})