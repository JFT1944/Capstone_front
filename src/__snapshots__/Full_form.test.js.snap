// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <form
    class="iForms"
    style="transform: translateY(50px);"
  >
    <h2>
      Copy And Paste Ingredients Here
    </h2>
    <span
      style="align-items: center;"
    >
      <span
        style="text-align: left; width: 40vw; min-width: 275px;"
      >
        Recipe Name (will not save recipe if left empty)
      </span>
      <input
        name="recipe_name"
        placeholder="Recipe Name (Not Necessary)"
        type="text"
        value=""
      />
    </span>
    <span
      style="align-items: center;"
    >
      <span
        style="text-align: left; width: 40vw; min-width: 275px;"
      >
        Recipe Ingredients
      </span>
      <textarea
        name="Ingredients"
      />
    </span>
    <span
      class="submit_button"
      style="align-items: center;"
    >
      <input
        type="submit"
      />
    </span>
  </form>
</DocumentFragment>
`;
